example_deck.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import plotly.express as px
+from flytekitplugins.deck.renderer import BoxRenderer, MarkdownRenderer
+from typing_extensions import Annotated
+
+import flytekit
+from flytekit import FlyteContextManager, dynamic, task, workflow
+from flytekit.deck.renderer import TopFrameRenderer
+
+iris_df = px.data.iris()
+
+
+@task()
+def t1() -> str:
+    md_text = "#Hello Flyte\n##Hello Flyte\n###Hello Flyte"
+    m = MarkdownRenderer()
+    s = BoxRenderer("sepal_length")
+    deck = flytekit.Deck("demo", s.to_html(iris_df))
+    deck.append(m.to_html(md_text))
+    default_deck = FlyteContextManager.current_context().user_space_params.default_deck
```

Comment:
```suggestion
    default_deck = flytekit.current_context().default_deck
```

Reply:
should be the same object as what you have.

Reply:
okay, thanks. will remove this file `example.py` before we merge this file